### DISCUSSION TITLE
Help with getting file to be in valid format

### DISCUSSION THREAD

**Discussion by alecjprobst** [svg2mod/svg2mod#56] [General]
I'm using the online version of the tool and it keeps telling me that `This does not appear to be a valid Inkscape SVG file. Note that this software is particular to the normal (not compressed) Inkscape format.` I have triple checked everything and I have the right layer names, I've converted everything that needed conversion, and I'm using the right file format (Inkscape SVG, not compressed). Does anyone know what might be wrong to cause this issue?
![Design_Prep](https://user-images.githubusercontent.com/49660264/213894590-23606347-2771-43b4-9610-32725284a460.svg)

**Comment by ithinkido** [comment]
Looks like the online version is broken. It works when usinhg self hosted .
[git.kicad_mod.zip](https://github.com/svg2mod/svg2mod/files/10474478/git.kicad_mod.zip)
